Sources/Core/Private/Resolver.swift

```diff
@@ -150,7 +150,7 @@ class Resolver {
     if hasMany {
       let filterComponents = components.filter{ !stack.contains($0.info) } // Remove objects contains in stack
       assert(nil == object, "Many injection not supported")
-      return filterComponents.flatMap{ makeObject(by: $0, use: nil) }
+		return filterComponents.compactMap{ makeObject(by: $0, use: nil) }
```

Comment:
#if swift(>= 4.1)
#else
#endif
maybe for ImplicitlyUnwrappedOptional need use 'if'

Reply:
Fixed